dpti/hti_water.py

```diff
@@ -891,5 +902,9 @@ def handle_compute(args):
     return info
 
 
+def handle_run(args):
```

Comment:
The `handle_run` function seems to be duplicated in multiple files. This is similar to the `run_task` function and should be refactored to avoid code duplication. [important]